src/components/v5/shared/RichText/hooks.ts

```diff
@@ -17,8 +16,10 @@ export const useRichText = (
   name: string,
   isDecriptionFieldExpanded: boolean,
 ) => {
-  const [content, setContent] = useState<string>('');
   const [notFormattedContent, setNotFormattedContent] = useState<string>('');
+  const { field } = useController({
```

Comment:
Why are you adding form-related hooks here?

Reply:
It was used before, I just move it up. We need the field value to display it as content.